dashboard/config/webpack.config.prod.js

```diff
@@ -144,7 +147,9 @@ module.exports = {
           // Process JS with Babel.
           {
             test: /\.(js|jsx)$/,
-            include: paths.appSrc,
+            include(file) {
+              return file.indexOf(paths.appSrc) === 0 || packageHasESNext(file);
+            },
```

Comment:
This is great.

Can we move these changes into a separate PR? It's seemingly tangential to the changes and with squash and merge we will partially lose context.

Reply:
Absolutely. Good call.